ui/build

```diff
@@ -34,12 +31,10 @@ build() {
   cd -
 }
 
-if [ -f $prll_sh ]; then # parallel execution!
-  export PRLL_NRJOBS="${PRLL_NRJOBS-10}"
-  echo "Building up to $PRLL_NRJOBS in parallel!"
-  . $prll_sh
-  prll build_ts $ts_apps
-  prll build $apps
+if command -v parallel; then # parallel execution!
+  export -f build_ts build
+  parallel -j+2 --bar build_ts ::: $ts_apps
```

Comment:
re -j+2 the justification is that some parts of the script are network bound not cpu bound